test/e2e/platform_operators_test.go

```diff
@@ -159,7 +159,7 @@ var _ = Describe("platform operators controller", func() {
 					if err := c.Get(ctx, types.NamespacedName{Name: po.GetName()}, bi); err != nil {
 						return nil, err
 					}
-					if bi.Status.InstalledBundleName == "" {
+					if bi.Status.ActiveBundle == "" {
```

Comment:
Is this supposed to come along?

Reply:
Yep, but I had sneaked it in. My IDE was complaining about it locally.

Reply:
Yes, I think it's ok -- we changed the name of that field upstream and it probably never got updated in the e2e test on the PO side.